File: logger/Archive/Tests/MicropythonBase/adc.py
from machine import ADC
import time

adc = ADC(26)  # GPIO26 = ADC0

while True:
    raw = adc.read_u16()  # 0..65535 (skalowane z 12-bit)
    print("ADC:", raw)
    time.sleep(0.2)

File: logger/Archive/Tests/PicoSDKBase/adc.c
#include "pico/stdlib.h"
#include "hardware/adc.h"

int main() {
    stdio_init_all();

    adc_init();
    adc_gpio_init(26);      // GPIO26 -> ADC0
    adc_select_input(0);    // kanał ADC0

    while (true) {
        uint16_t raw = adc_read(); // 12-bit: 0..4095
        // np. prosto: mrugaj LED zależnie od progu
        sleep_ms(200);
        (void)raw;
    }
}